### ISSUE TITLE
Remove Custom Pages dependency

### ISSUE THREAD

**Issue by CarlSinclair** [humhub-contrib/blog#4]
Custom Pages is a pretty niche module. Blog is not. Blog should work as long as it's enabled globally, even if it's not enabled in the space

**Comment by buddh4** [comment]
Yeah would be nice, perhaps the custom pages module should allow other modules to add pages without the module itself beeing installed on a space. Full indepence of the custom pages module is not planned.
